WooCommerce/WooCommerceTests/ViewRelated/Products/Media/ProductUIImageLoaderTests.swift

```diff
@@ -27,7 +27,7 @@ final class ProductImagesProviderTests: XCTestCase {
 
     func testRequestingImageWithRemoteProductImage() {
         let mockPHAssetImageLoader = MockPHAssetImageLoader(imagesByAsset: [:])
-        let imagesProvider = DefaultProductImagesProvider(imageService: imageService, phAssetImageLoader: mockPHAssetImageLoader)
+        let imagesProvider = DefaultProductUIImageLoader(imageService: imageService, phAssetImageLoader: mockPHAssetImageLoader)
```

Comment:
We missed this one. 🙂 

```suggestion
        let imageLoader = DefaultProductUIImageLoader(imageService: imageService, phAssetImageLoader: mockPHAssetImageLoader)
```

Reply:
yup! updated in: https://github.com/woocommerce/woocommerce-ios/pull/1914/commits/bef63d1cf1e24112f0cbf95be40b67d224a16658